### ISSUE TITLE
Squiz.Formatting.OperatorBracket autofix incorrect when assignment used with null coalescing operator

### ISSUE THREAD

**Issue by PaulBehrendtVentoro** [squizlabs/PHP_CodeSniffer#3003]
Hi,

We added `Squiz.Formatting.OperatorBracket` to our ruleset and the autofix operation did this:

```
-        return $this->gmail
-            ?? $this->gmail = new Google_Service_Gmail($this->google);
+        return ($this->gmail
+            ?? $this->gmail) = new Google_Service_Gmail($this->google);
```

Using v3.5.5

Thanks for help!

**Comment by gsherwood** [comment]
Thanks for this report. I've finally managed to get around to fixing it. The fix will be released in 3.6.0.

**Comment by hieu97-cry** [comment]
Thank you very muck!!!

Vào 12:36, T.2, 18 Th1, 2021 Greg Sherwood <notifications@github.com> đã
viết:
